### PR TITLE
Updating workshop request form

### LINKED ISSUE #1076: Update form to request workshop
http://software-carpentry.org/workshops/request.html need to be update to reflect our new fee, http://software-carpentry.org/blog/2015/07/changes-to-admin-fee.html.

### PR DESCRIPTION
**Pull request by gvwilson**
Closes #1076.

### CODE REVIEW

File: workshops/request.html [hunk 1]
```diff
@@ -5,17 +5,28 @@
...
+<p>Our instructors are volunteers, and so are not paid for their teaching, but host sites are required to cover their travel and accommodation costs. In addition, the Software Carpentry Foundation charges an administration fee for workshops that it helps organize at institutions which are not partners or affiliates.  For workshop requests received after September 1, 2015, these fees are:</p>
 <ul>
   <li>
-    <p>If you are a not-for-profit, such as a university or government lab, this is presently US$1250 for the first worskhop at an institution in any given year, and US$750 for each subsequent workshop in that year at that institution.</p>
+    <p>If you are a not-for-profit, such as a university or government lab: US$2500 for the first worskhop at an institution in any given year, and US$1250 for each subsequent workshop in that year at that institution.</p>
```
Comment (rgaiacs) [inline workshops/request.html]: US$2500 per workshop. If partner of affiliate that request our help, US$12500.

File: workshops/request.html [hunk 2]
```diff
@@ -5,17 +5,28 @@
...
   </li>
   <li>
-    <p>If you are a for-profit institution, such as a company, the fees are US$5000 and US$3000 respectively, of which three quarters is used to underwrite workshops at institutions that could otherwise not afford them.</p>
+    <p>If you are a for-profit institution, such as a company: US$10,000 and US$6000 respectively, of which three quarters is used to underwrite workshops at institutions that could otherwise not afford them.</p>
```
Comment (rgaiacs) [inline workshops/request.html]: US$10,000 per workshop.

File: workshops/request.html [hunk 3]
```diff
@@ -5,17 +5,28 @@
...
   </li>
 </ul>
 <p>
-  Workshops requested from September 1, 2015 on will be subject to <a href="{{page.root}}/blog/2015/07/changes-to-admin-fee.html">a new fee schedule</a>.
+  As explained in <a href="{{page.root}}/blog/2015/07/changes-to-admin-fee.html">this blog post</a>,
```
Comment (rgaiacs) [inline workshops/request.html]: Maybe

> If you organized one workshop before September 1, 2015 you should check <a href="{{page.root}}/blog/2015/07/changes-to-admin-fee.html">this blog post</a> (...)

File: workshops/request.html [hunk 4]
```diff
@@ -174,14 +185,14 @@
...
-<span class="ss-choice-label">For-profit: full fee for first workshop/year (US$5000)</span>
-</label></li> <li class="ss-choice-item"><label><span class="ss-choice-item-control goog-inline-block"><input type="radio" name="entry.793641257" value="For profit: full fee for subsequent workshop/year (US$3000)" id="group_793641257_4" role="radio" class="ss-q-radio" aria-label="For profit: full fee for subsequent workshop/year (US$3000)" required="" aria-required="true"></span>
-<span class="ss-choice-label">For profit: full fee for subsequent workshop/year (US$3000)</span>
+<ul class="ss-choices" role="radiogroup" aria-label="Which of the following applies to your payment for the administrative fee?  "><li class="ss-choice-item"><label><span class="ss-choice-item-control goog-inline-block"><input type="radio" name="entry.793641257" value="Non-profit: full fee for first workshop/year (US$2500)" id="group_793641257_1" role="radio" class="ss-q-radio" aria-label="Non-profit: full fee for first workshop/year (US$2500)" required="" aria-required="true"></span>
+<span class="ss-choice-label">Non-profit: full fee for first workshop/year (US$2500)</span>
```
Comment (rgaiacs) [inline workshops/request.html]: "Non-profit: full fee for first workshop/year" -> "Non-profit: full fee for one workshop"

File: workshops/request.html [hunk 5]
```diff
@@ -174,14 +185,14 @@
...
+</label></li> <li class="ss-choice-item"><label><span class="ss-choice-item-control goog-inline-block"><input type="radio" name="entry.793641257" value="Non-profit: reduced fee for subsequent workshop/year (US$1250)" id="group_793641257_2" role="radio" class="ss-q-radio" aria-label="Non-profit: reduced fee for subsequent workshop/year (US$1250)" required="" aria-required="true"></span>
+<span class="ss-choice-label">Non-profit: reduced fee for subsequent workshop/year (US$1250)</span>
```
Comment (rgaiacs) [inline workshops/request.html]: Drop this.

File: workshops/request.html [hunk 6]
```diff
@@ -174,14 +185,14 @@
...
+</label></li> <li class="ss-choice-item"><label><span class="ss-choice-item-control goog-inline-block"><input type="radio" name="entry.793641257" value="For-profit: full fee for first workshop/year (US$10,000)" id="group_793641257_3" role="radio" class="ss-q-radio" aria-label="For-profit: full fee for first workshop/year (US$10,000)" required="" aria-required="true"></span>
+<span class="ss-choice-label">For-profit: full fee for first workshop/year (US$10,000)</span>
```
Comment (rgaiacs) [inline workshops/request.html]: "For-profit: full fee for first workshop/year (US$10,000)" -> "For-profit: full fee for one workshop (US$10,000)"

File: workshops/request.html [hunk 7]
```diff
@@ -174,14 +185,14 @@
...
+</label></li> <li class="ss-choice-item"><label><span class="ss-choice-item-control goog-inline-block"><input type="radio" name="entry.793641257" value="For profit: full fee for subsequent workshop/year (US$6000)" id="group_793641257_4" role="radio" class="ss-q-radio" aria-label="For profit: full fee for subsequent workshop/year (US$6000)" required="" aria-required="true"></span>
+<span class="ss-choice-label">For profit: full fee for subsequent workshop/year (US$6000)</span>
```
Comment (rgaiacs) [inline workshops/request.html]: Drop this.

File: workshops/request.html [hunk 8]
```diff
@@ -174,14 +185,14 @@
...
 </label></li> <li class="ss-choice-item"><label><span class="ss-choice-item-control goog-inline-block"><input type="radio" name="entry.793641257" value="No fee (my organization is a Partner or Affiliate)" id="group_793641257_5" role="radio" class="ss-q-radio" aria-label="No fee (my organization is a Partner or Affiliate)" required="" aria-required="true"></span>
 <span class="ss-choice-label">No fee (my organization is a Partner or Affiliate)</span>
```
Comment (rgaiacs) [inline workshops/request.html]: Add "Partner of Affiliate: special fee for one workshop (US$12500)" and reword this to "No fee (self-organized workshop)".

**Comment by gvwilson** [comment]
@rgaiacs thanks for catching all that - please let me know how this looks.
